src/mongocxx/private/libmongoc_symbols.hh

```diff
@@ -277,7 +277,7 @@ MONGOCXX_LIBMONGOC_SYMBOL(server_api_get_strict)
 MONGOCXX_LIBMONGOC_SYMBOL(server_api_get_version)
 MONGOCXX_LIBMONGOC_SYMBOL(server_description_host)
 MONGOCXX_LIBMONGOC_SYMBOL(server_description_id)
-MONGOCXX_LIBMONGOC_SYMBOL(server_description_ismaster)
+MONGOCXX_LIBMONGOC_SYMBOL(server_description_hello_response)
```

Comment:
`server_description_ismaster` is now deprecated (CDRIVER-3947) and will cause build failures, so I've updated usages to `server_description_hello_response`.